core/src/main/java/com/klaytn/caver/utils/Utils.java

```diff
@@ -411,7 +432,51 @@ public static String recover(String message, SignatureData signatureData, boolea
         if (key == null) {
             throw new SignatureException("Could not recover public key from signature");
         }
-        return Numeric.prependHexPrefix(Keys.getAddress(key));
+
+        return Numeric.toHexStringWithPrefixZeroPadded(key, LENGTH_PUBLIC_KEY_STRING_DECOMPRESSED);
+    }
+
+    /**
+     * Returns an address which is derived by a public key(handles both compressed format and decompressed format).<p>
+     * This function simply converts the public key string into address from by hashing it.<p>
+     * It has nothing to do with the actual account in the Klaytn.
+     * <pre>Example :
+     * {@code
+     * String address = caver.utils.publicKeyToAddress("0x{public key}");
+     * }
+     * </pre>
+     * @param publicKey The public key string to get the address.
+     * @return String
+     */
+    public static String publicKeyToAddress(String publicKey) {
+        publicKey = Utils.decompressPublicKey(publicKey);
+        return addHexPrefix(Keys.getAddress(publicKey));
+    }
+
+    /**
+     * Recovers the address that was used to sign the given data.
+     * This function automatically creates a message hash by appending a Klaytn sign prefix to the message.
+     * @param message A plain message when using signed.
+     * @param signatureData The signature values in KlaySignatureData
+     * @return String
+     * @throws SignatureException It throws when recover operation has failed.
+     */
+    public static String recover(String message, SignatureData signatureData) throws SignatureException {
+        return recover(message, signatureData, false);
+    }
+
+
+    /**
+     * Recovers the address that was used to sign the given data.
+     * @param message A plain message or hashed message.
+     * @param signatureData The signature values in KlaySignatureData
+     * @param isPrefixed If true, the message param already hashed by appending a Klaytn sign prefix to the message.
+     * @return String
+     * @throws SignatureException It throws when recover operation has failed.
+     */
+    public static String recover(String message, SignatureData signatureData, boolean isPrefixed) throws SignatureException {
```

Comment:
```suggestion
    public static String recover(String message, SignatureData signatureData, boolean isHashed) throws SignatureException {
```
Can you change last parameter name of `recover` functions?